contribs/gmf/less/mobile.less

```diff
@@ -253,3 +253,20 @@ button[ngeo-mobile-geolocation] {
     }
   }
 }
+
+//mobile styles for the theme and background selector
+.gmf-theme-selector,
+.gmf-mobile-background-layer-selector {
+  .gmf-icon-check:after {
+    font-size: 1.2rem;
+    position: relative;
+    bottom: 0.8rem;
+  }
+  .gmf-text {
+    margin: 3% 10px 0 0;
+  }
+  .gmf-thumb {
+    height: auto;
+    width: 25%;
```

Comment:
@sbrunner: Because the images are too big and would look like this if we do not adjust the CSS:
![screen shot 2016-02-18 at 17 44 49](https://cloud.githubusercontent.com/assets/1745041/13150726/4935ae58-d667-11e5-9b21-c2028a44c954.png)